pkg/deploy/registry/devfileregistry.go

```diff
@@ -0,0 +1,104 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+package registry
```

Comment:
maybe we should rename file devfileregistry.go  -> devfile_registry.go , f.e https://github.com/golang/go/blob/master/src/crypto/subtle/constant_time.go